### ISSUE TITLE
Version 184.0 not buildable

### ISSUE THREAD

**Issue by berendt** [gardenlinux/gardenlinux#164]
54dbe2db129e9ba063916f4d10763b06cd30a660 is missing.

```
gpg: directory '/home/runner/.gnupg' created
gpg: keybox '/home/runner/.gnupg/pubring.kbx' created
gpg: /home/runner/.gnupg/trustdb.gpg: trustdb created
gpg: error reading key: No secret key
No secret key for contact@gardenlinux.io exists, signing disabled
/bin/sh: 1: cannot create .build/sign.pub: Directory nonexistent
make: *** [sign.pub] Error 2
Makefile:10: recipe for target 'sign.pub' failed
```

**Comment by marwinski** [comment]
I have cherry picked the fix from the main branch to rel-184: https://github.com/gardenlinux/gardenlinux/commit/3cfe8daefb207c50cca26b3fe0d0da9db6aa0e07

**Comment by marwinski** [comment]
Build fails but for a different reason now. I will investigate.

**Comment by marwinski** [comment]
/assign @gehoern 

I assume this is related to moving to a proper repository location!? What do we need to cherry-pick ?

**Comment by gehoern** [comment]
/assign @ccwienk 

problem is broken VERSION file in 184 ... contents should be 184 NOT 1.184

**Comment by marwinski** [comment]
This is caused by a wrong version in the VERSION file. It is 1.184 but should be 184.0

**Comment by fynluk** [comment]
Build still fails (make openstack) while installing dracut modules:
```
dracut-install: ERROR: installing 'growpart'
dracut: FAILED: /usr/lib/dracut/dracut-install -D /var/tmp/dracut.Xk3jOb/initramfs -a grep sfdisk growpart udevadm awk mawk sed rm readlink

**Comment by 5kt** [comment]
This has been fixed by : [71883cad58d4892306e4239a1f483a27f9f916b9](https://github.com/gardenlinux/gardenlinux/commit/71883cad58d4892306e4239a1f483a27f9f916b9) and [5e5cd88280a951c5d4cb4a7e5e48d4f0f619ac0d](https://github.com/gardenlinux/gardenlinux/commit/5e5cd88280a951c5d4cb4a7e5e48d4f0f619ac0d)

This was caused by the missing cloud-guest-utils package that was getting removed when libdb is uninstalled - dependency issues.

**Comment by marwinski** [comment]
Thanks @5kt. I will close this issue.

/close
